### DISCUSSION TITLE
Add track slider

### DISCUSSION THREAD

**Discussion by JordanTranchina** [fabiusBile/Spotify4BigSur#4] [Ideas]
Hi @fabiusBile! Any chance you could add a track slider in the widget just like the old controller (in the first screenshot above). Love the app!

**Comment by fabiusBile** [comment]
Hi! Unfortunately, WidgetKit doesn’t allow any interactive elements, other than simple buttons (a.k.a links), so it’s impossible to create a slider 😔
